README_ESPRESSO.md

```diff
@@ -209,6 +209,45 @@ nix --extra-experimental-features "nix-command flakes" develop
 just espresso-enclave-tests
 ```
 
+#### Building, running and registering enclave images
+
+`op-batcher/enclave-tools` provides a command-line utility for common operations on batcher enclave images.
+Before using it, set your AWS instance as described in the guide above, then build the tool:
+
+```
+cd op-batcher/
+just enclave-tools
+```
+
+This should create `op-batcher/bin/enclave-tools` binary. You can run
+```
+./op-batcher/bin/enclave-tools --help
+```
+to get information on available commands and flags.
+
+##### Building a batcher image
+
+To build a batcher enclave image, and tag it with specified tag:
+```
+./op-batcher/bin/enclave-tools build --op-root ./ --tag op-batcher-enclave
```

Comment:
A bit like the question above. In practice is this tag an url, something like
"ghcr.io/espressosystems/espresso-sequencer/espresso-dev-node:release-colorful-snake", or just  "release-colorful-snake"

Reply:
I tried and just running the command as is works.